packages/react-components/src/box/src/Box.tsx

```diff
@@ -0,0 +1,46 @@
+import { ComponentProps, ElementType, ForwardedRef, ReactElement, ReactNode } from "react";
+import { forwardRef, omitProps } from "../../shared";
+
+export interface InnerBoxProps {
+    /**
+    * An HTML element type or a custom React element type to render as.
+    */
+    as?: ElementType;
+    /**
+     * [Slot](?path=/docs/getting-started-slots--page) to render into.
+     */
+    slot?: string;
+    /**
+     * React children
+    */
+    children?: ReactNode;
```

Comment:
Add @ignore since as discussed we only want to document required children for now.